docs/home/feature-management/features-and-variables/variables-and-variations.md

```diff
@@ -58,6 +58,30 @@ Taking this action will cause all references to the Variable in any code usage t
 
 To fully delete a Variable you must do so via our [Management API](/management-api/#operation/VariablesController_remove).
 
+
+### Re-associating a Variable
+DevCycle has the ability to re-use existing variables and re-associate them to different features. 
+
+In the Variable Key input field, a drop down will display all **unassociated, unarchived** variables that can be re-associated to your feature while also providing you the option to add a net new variable.
+
+![Add Variable Modal Dropdown](/april-2023-add-variable-dropdown.png)
+
+In the example above, the variables `demo` and `realtime-demo` are not associated with any other feature and can be added to my feature. 
+
+If you select an existing, unassociated variable from the dropdown, the Variable type will be populated with the type of the selected variable and cannot be changed. 
+
+![Add New Variable Variable Selected](/april-2023-add-new-variable-variable-selected-tooltip.png)
+
+If you input a variable key name with the same key as an existing variable, but this variable is archived. The error below will appear, as you must first unarchive the variable. 
+
+![Add New Variable Archived Variable Error](/april-2023-add-variable-modal-archived-variable-error.png)
+
+If you wish to use it, click on the hyperlinked "**variable**" text and it will take you directly to the archived variable page for you to unarchive. 
+
+:::info
+If you would like to move a variable between features, the variable must first be removed from the previous feature to fall into an unassociated state. This means that the variable will return back to it's default values for the duration it is not associated to a feature. 
```

Comment:
"If you would like to move a variable between features, the variable must first be removed from the previous feature to fall into an unassociated state." 

could be clearer as: 

"If you want to move a variable between features, you must first remove it from the previous feature, making it unassociated."